src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -92,11 +93,15 @@ class FormatManager implements FormatManagerInterface
      */
     private $formats;
 
+    /** @var VideoThumbnailServiceInterface */
+    private $videoThumbnailService;
+
     /**
      * @param MediaRepository         $mediaRepository
```

Comment:
I know it is not your fault, but still: can you reformat this file, so that the alignment in this PHPDoc is gone?